Makefile

```diff
@@ -302,27 +350,26 @@ socketcan:
 	@sudo ip link set up vcan0 || true
 	@ip link show vcan0
 
-.PHONY: update_codegen
-update_codegen:
-	@python make/git_fetch.py -folder=libraries/codegen-tooling -user=uw-midsun -repo=codegen-tooling-msxiv -tag=latest -file=codegen-tooling-out.zip
-
 .PHONY: pytest
 pytest:
 	@python3 -m unittest discover -t $(PROJ_DIR)/$(PROJECT)/scripts -s $(PROJ_DIR)/$(PROJECT)/scripts -p "test_*$(TEST).py"
 
 .PHONY: pytest_all
-pytest_all:
-	@for i in $$(find projects -name "test_*.py"); 													\
+pytest_all: codegen_test
+	@for i in $$(find . -path ./.venv -prune -false -o -name "test_*.py"); 													\
```

Comment:
An issue I didn't foresee: this also runs all the MPXE integration tests since they match the `test_*.py` glob. Could you prune the MPXE integration test dir too?

Also, you shouldn't need the dependency on `codegen_test`.